wpilibc/src/generated/main/native/include/frc/PS5Controller.h

```diff
@@ -478,38 +491,62 @@ class PS5Controller : public GenericHID {
    *
    * @return The state of the button.
    */
-  bool GetTouchpad() const;
+  bool GetTouchpadButton() const;
 
   /**
-   * Whether the touchpad was pressed since the last check.
+   * Whether the touchpad button was pressed since the last check.
    *
-   * @return Whether the touchpad was pressed since the last check.
+   * @return Whether the button was pressed since the last check.
    */
-  bool GetTouchpadPressed();
+  bool GetTouchpadButtonPressed();
 
   /**
-   * Whether the touchpad was released since the last check.
+   * Whether the touchpad button was released since the last check.
    *
-   * @return Whether the touchpad was released since the last check.
+   * @return Whether the button was released since the last check.
    */
-  bool GetTouchpadReleased();
+  bool GetTouchpadButtonReleased();
 
   /**
-   * Constructs an event instance around the touchpad's digital signal.
+   * Constructs an event instance around the touchpad button's
+   * digital signal.
    *
    * @param loop the event loop instance to attach the event to.
-   * @return an event instance representing the touchpad's digital signal
-   * attached to the given loop.
+   * @return an event instance representing the touchpad button's
+   * digital signal attached to the given loop.
    */
   BooleanEvent Touchpad(EventLoop* loop) const;
 
+
+
```

Comment:
Are these blank lines worth worrying about? I don't think so, but I want to make sure.